src/colorizer/shaders/line.vert

```diff
@@ -0,0 +1,15 @@
+out vec2 vUv;
+
+uniform highp usampler2D frame;
+uniform float aspect;
+
+void main() {
+    // Apply scale to account for frame scaling in the canvas
+    // (copied from `colorize_RGBA8U.frag`)
+    ivec2 frameDims = textureSize(frame, 0);
+    float frameAspect = float(frameDims.x) / float(frameDims.y);
+    vec2 scale = max(vec2(aspect / frameAspect, frameAspect / aspect), 1.0);
```

Comment:
what's the difference between aspect and frameAspect?

Reply:
`aspect` is the aspect of the dataset, `frameAspect` is the aspect of the canvas. These lines are copied over from the colorize shader and are for fitting the image into the canvas with the correct aspect ratio. Currently the canvas has the same aspect as all our datasets, but if it didn't this would shrink to fit.

It's possible that in both cases this bit of vector math could be done outside the shader?

Reply:
I see. Until today I didn't realize the canvas could ever be a different aspect ratio than the actual image data. Couldn't we just resize the canvas to always keep that in sync?

Reply:
anyway, knowing this, I would say the vertex shader is totally fine.  I'd save the change I mention above for a separate PR.

Reply:
We could, but I figured the problem of keeping a canvas at the right aspect ratio in the context of a webpage (where it may have to share space and play nice with other elements) would be tougher than keeping a texture at the right aspect ratio within a canvas. Peyton and I were talking about just moving this vector math out of the shader, as I suggested above. I think either one would free up the path to use Three's native `LineBasicMaterial` rather than messing with custom shaders, which feels like a worthwhile thing to do in this PR if possible.

Reply:
Done! I made the calculations for this a bit more verbose but with more clarity. I also added Cameron's suggestion of doing the calculation outside the shader and passing it in.